scripts/autotools/libmongoc/CheckSSL.m4

```diff
@@ -12,7 +12,14 @@ PHP_ARG_WITH([openssl-dir],
              [auto],
              [no])
 
+AS_IF([test "$PHP_MONGODB_SSL" = "yes"],[
```

Comment:
Would it make sense to add a comment explaining why we handle "yes" even though it's not a documented option. Something like:

> PHP_ARG_WITH will substitute "yes" in the absence of a value. Treat it like "auto" but required.

Likewise for CheckSasl.m4 (above the first place you reference "yes" as a value).